### ISSUE TITLE
goa v1 to v3

### ISSUE THREAD

**Issue by phanirithvij** [ncarlier/feedpushr#93]
Is this possible?

**Comment by ncarlier** [comment]
Yes it is and I would love to do that. I "just" need to find some time 😅

**Comment by mszmidt** [comment]
Hi, also interested, when I try to `go get -u`

```go
...
go: github.com/ncarlier/feedpushr/v3 imports
	github.com/ncarlier/feedpushr/v3/pkg/server imports
	github.com/ncarlier/feedpushr/v3/autogen/app imports
	github.com/goadesign/goa/cors: cannot find module providing package github.com/goadesign/goa/cors
...
```

apparently when I `go get -u` without goa it allows to update 'em

I'm afraid that because it is stick it **may** also cause some security issues. My scanner reports that:
- [CVE-2024-45337](https://avd.aquasec.com/nvd/cve-2024-45337) (golang/crypto)
- [CVE-2022-41723](https://avd.aquasec.com/nvd/cve-2022-41723) (golang/net)
- [CVE-2023-39325](https://avd.aquasec.com/nvd/cve-2023-39325) (golang/net)
- [CVE-2024-45338](https://avd.aquasec.com/nvd/cve-2024-45338) (golang/net)

golang/crypto seems to be fixed in at least 0.31.0
golang/net seems to be fixed in 0.33.0

dunno which goa version has those fixed
